### PR TITLE
A short editorial review of disaster recovery page, update the misleading steps, and update the error message

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1547.surge.sh

Updated pages:
- [Disaster recovery](https://neo4j-docs-operations-1547.surge.sh/operations-manual/5/clustering/disaster-recovery)

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 1]
```diff
@@ -3,21 +3,18 @@
...
-By performing the actions described here, the unavailable databases are recovered and made fully operational with as little impact as possible on the other databases in the cluster.
+A database can become unavailable due to issues on different system levels.
+For example, a data centre failover may lead to the loss of multiple servers, which may cause a set of databases to become unavailable.
+It is also possible for databases to become quarantined due to a critical failure in the system, which may lead to unavailability even without the loss of servers.
 
-There are many reasons why a database becomes unavailable and it can be caused by issues on different levels in the system.
-For example, a data-center failover may lead to the loss of multiple serves which in turn may cause a set of databases to become unavailable.
-It is also possible for databases to become quarantined due to a critical failure in the system which may lead to unavailability even without loss of servers.
+This section contains a step-by-step guide on how to recover databases that have become unavailable in terms of write operations, while still being able to serve read operations.
```
Comment (jackwaudby) [inline modules/ROOT/pages/clustering/disaster-recovery.adoc:10]: I like the clarify of:
> an _unavailable database_ is defined as a database that is incapable of serving writes

Could this be worked into the new text?

**Review by jackwaudby — APPROVED** [review]

File: modules/ROOT/pages/clustering/disaster-recovery.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:10]: Thanks for the quick review, @jackwaudby. Which text do you refer to?

Comment (renetapopova) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:10]: Ah, I think I got it.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1547.surge.sh

Updated pages:
- [Disaster recovery](https://neo4j-docs-operations-1547.surge.sh/operations-manual/5/clustering/disaster-recovery)

**Review by jackwaudby — APPROVED** [review]

**Review by NataliaIvakina — APPROVED** [review]
Good start! Thank you, @renetapopova

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 2]
```diff
@@ -31,22 +28,21 @@ Consequently, in a disaster where multiple servers go down, some databases may k
...
+The `system` database defines the configuration for the other databases, and therefore, it is vital to ensure that it is available before doing anything else.
 
-The first step is to ensure that the `system` database is available in the cluster.
-The `system` database defines the configuration for the other databases and therefore it is vital to ensure that it is available before doing anything else.
+Once the `system` database's availability is verified, whether recovered or unaffected by the disaster, the next step is to recover lost servers to ensure the cluster's topology meets the requirements.
 
-Once the `system` database's availability is verified, whether it was recovered or unaffected by the disaster, the next step is to recover lost servers to make sure the cluster's topology requirements are met.
-
-Only after the `system` database is available and the cluster topology is satisfied, can the databases be managed.
+Only after the `system` database is available and the cluster topology is satisfied can the databases be managed.
```
[the remark this answers is not in the document]
Comment (jackwaudby) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:39]: @NataliaIvakina can only -> can then

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1547.surge.sh

Updated pages:
- [Disaster recovery](https://neo4j-docs-operations-1547.surge.sh/operations-manual/5/clustering/disaster-recovery)

**Pull request merged**